cloco_cli/cli.py

```diff
@@ -168,6 +168,111 @@ def delete_subscription_permission(sub, username):
     return
 
 
+@subscription.group('client')
+def subscription_clients():
+    """A subgroup of commands for managing subscription clients"""
+    return
+
+
+@subscription_clients.command('list')
+@click.option('--sub', help='The subscription identifier, will use the subscription stored in the preferences if not supplied', default='')
+def list_subscription_client(sub):
+    """Returns a list of the subscription clients"""
+    config = load_config()
+    authenticate(config)
+    if not sub:
+        sub = config['preferences']['subscription']
+    u = '{0}/{1}/clients'.format(get_url(config), sub)
+    r = requests.get(u, headers=get_headers(config))
+    print_json_response(r)
```

Comment:
more of a general question, response contain sensitive things like api key or somesuch when it dumps to stdout?